Makefile

```diff
@@ -1,28 +1,49 @@
 .PHONY: test test-coverage test-coverage-html test-coverage-clover test-unit test-unit-coverage test-unit-coverage-html test-unit-coverage-clover install
 
-test:
-	@./vendor/bin/phpunit
+# Setting up
 
-test-coverage:
-	@./vendor/bin/phpunit --coverage-text
+setup: ## Install dependencies and set up example conf file
+	@docker-compose run --rm composer install
+
+# Testing
+
+test: ## Run all tests
+test: test-coverage test-coverage-html test-coverage-clover test-unit test-unit-coverage test-unit-coverage-html test-unit-coverage-clover
```

Comment:
There's repeated build targets here. Also, you don't need to do all three versions of coverage. It's the same thing but just different output.